app/Http/Controllers/TasksController.php

```diff
@@ -0,0 +1,61 @@
+<?php
+
+namespace App\Http\Controllers;
+
+use App\Task;
+use Illuminate\Http\Request;
+
+class TasksController extends Controller
+{
+
+    public function __construct()
+    {
+        $this->middleware('auth');
+    }
+    public function index()
+    {
+
+    }
+
+    public function store(Request $request)
+    {
+        $task = new Task;
+        $task->content = $request->content;
+        $task->project_id = $request->project_id;
+        $task->order = $request->order;
```

Comment:
do we need to calculate order on fronted, why we dont do it in model?
user shouldn't wait  while you are calculating your/his FUTURE data on front